lib/shared/config-manager/src/index.ts

```diff
@@ -63,28 +84,121 @@ export class EnvironmentConfigManager {
                 this.logger.debug('DevCycle initial config loaded')
             })
             .finally(() => {
-                if (this.disablePolling) {
-                    return
-                }
-                this.intervalTimeout = this.setInterval(async () => {
-                    try {
-                        await this._fetchConfig()
-                    } catch (ex) {
-                        this.logger.error((ex as Error).message)
-                    }
-                }, this.pollingIntervalMS)
+                this.startPolling(this.configPollingIntervalMS)
+                this.startSSE()
```

Comment:
why do you need a special case of starting SSE here rather than just letting the existing config fetch handler do it?

Reply:
good question, I need to double-check that.

Reply:
I think it makes sense to keep starting SSE + Polling here, as it ensures that polling is started the first time before SSE is started, and that the polling code doesn't need to understand if SSE is connected or not. The SSE code will handle changing the polling interval once it has connected.